scripts/collect_release_benchmarks.py

```
@@ -0,0 +1,387 @@
+
+
+def _venv_python(venv_path: Path) -> Path:
+    match platform.system():
+        case "Windows":
+            return venv_path / "Scripts" / "python.exe"
+        case _:
+            return venv_path / "bin" / "python"
+
+
+def _run_subprocess(command: list[str], *, timeout: int, cwd: Path = ROOT) -> subprocess.CompletedProcess[str]:
+    return subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)